src/test/java/com/st/lms/LmsSpring/LmsSpringApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.st.lms.LmsSpring;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class LmsSpringApplicationTests {
```

Comment:
1. Your PR should include comprehensive tests.

2. Any "test" class that doesn't include any test *code* should be removed.